dmscripts/upload_counterpart_agreements.py

```diff
@@ -0,0 +1,46 @@
+import getpass
+
+from dmutils.documents import generate_timestamped_document_upload_path, generate_download_filename, \
+    COUNTERPART_FILENAME
+
+from dmscripts.bulk_upload_documents import get_supplier_id_from_framework_file_path
+
+
+def upload_counterpart_file(bucket, framework_slug, file_path, dry_run, client, logger):
```

Comment:
Oh dear I just started following the weird pattern with my script wanting to fit in... :)